mimic-iv/concepts_duckdb/README.md

```diff
@@ -0,0 +1,16 @@
+# duckdb concepts
+
+This folder has SQL compatible with [DuckDB](https://duckdb.org/).
+These concepts were generated automatically from the BigQuery SQL dialect using the [sqlglot](https://sqlglot.com/) package.
+If you would like to contribute a correction, it should be for the corresponding file in the concepts folder.
```

Comment:
I think the following sentence is saying that corrections should be made to the BigQuery SQL dialect? 

`If you would like to contribute a correction, it should be for the corresponding file in the concepts folder.`

If so (which I think is best), it might be worth making the statement more explicit (e.g. "in the BigQuery concepts folder").